nautobot/extras/models/change_logging.py

```diff
@@ -80,14 +80,14 @@ class ObjectChange(BaseModel):
         blank=True,
         null=True,
     )
-    user_name = models.CharField(max_length=150, editable=False, db_index=True)
+    user_name = models.CharField(max_length=255, editable=False, db_index=True)
```

Comment:
`User.username` is defined by Django (https://github.com/django/django/blob/stable/3.2.x/django/contrib/auth/models.py#L332) to a `max_length=150`, so we may as well stick to that pattern.